torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -1413,7 +1413,7 @@ class CppMicroGemmWoQInt4Avx512(CppMicroGemmFP32Vec):
     int64_t ldb,
     int64_t ldc,
     int64_t q_group_size,
-    const bfloat16* {{restrict_keyword}} ScaleAndZeros,
+    const at:BFloat16* {{restrict_keyword}} ScaleAndZeros,
```

Comment:
should this be `at::BFloat16`?

Reply:
Wooh, I am surprised this is not caught by any test...